trin-core/src/portalnet/types/messages.rs

```diff
@@ -129,6 +130,7 @@ impl Default for PortalnetConfig {
             data_radius: U256::from(u64::MAX), //TODO better data_radius default?
             no_stun: false,
             enable_metrics: false,
+            enr_cache_capacity: 256,
```

Comment:
I think 256 is a reasonable initial value. I guess we expect only the first 7-8 buckets to include any nodes, so if we take the max capacity of 16 nodes per bucket and 8 buckets x 2 networks = 256.